### DISCUSSION TITLE
elastic.out file

### DISCUSSION THREAD

**Discussion by jmarbues** [brucefan1983/GPUMD#751] [Q&A] [answered]
Hello. In your website there is the compute_elastic parameter that allows us to compute the elastic constants based on an applied strain. Could you add a page on the website for the elastic.out like you have for other output files?

Also, the symmetry type you have available is cubic. Does that mean that the volume is increased isotropically in all 3 directions?

Thank you for your time and help. Kind regards, José

**Comment by brucefan1983** [comment, marked as answer]
Hello, the `elastic.out` file just contains the calculated elastic constant values such as
```
C11 = 1133.53 Gpa
C12 = 153.033 GPa
C44 = 604.509 GPa
```

Currently we have only implemented cubic systems and have not motivated to do more because there are better ways to calculate the elastic constants. For example, you can check this tutorial in calorine:

https://calorine.materialsmodeling.org/tutorials/elastic_stiffness_tensor.html

About calorine:

calorine is a Python library for constructing and sampling neuroevolution potential (NEP) models via the [GPUMD](https://gpumd.org/) package. It provides ASE calculators, IO functions for reading and writing GPUMD input and output files, as well as a Python interface that allows inspection of NEP models.
